rock.osdeps

```diff
@@ -411,3 +411,12 @@ gawk:
 
 bzip2:
     debian,ubuntu: libbz2-dev
+
+slam/pcl:
+    ubuntu:
+        default: libpcl-dev
+        '14.04': libpcl-1.7-all-dev
```

Comment:
Sorry, my bad, ros is not added by sources.list, but by /etc/apt/sources.list.d/ros-latest.list